generator/Generator/ModelGenerator.php

```diff
@@ -141,7 +142,10 @@ public function generate(): void
                         continue;
                     }
 
+                    $newNamespaceDirectory = rtrim($newNamespaceDirectory, '\\');
```

Comment:
If there is no subnamespace/subdirectory, like for `OriginErrors`, this needs to trim the backslashes, else it will try to create a somewhat double directory structure.